lib/util/cmd.go

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+package util
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/adobe/aquarium-fish/lib/log"
+)
+
+// Runs & logs the executable command
+func RunAndLog(sec string, timeout time.Duration, stdin io.Reader, path string, arg ...string) (string, string, error) {
```

Comment:
The name of the first parameter `sec` got me pretty confused.  I originally thought it was supposed to mean `seconds`, but it doesn't, it's just a descriptor of the command, e.g., `"NATIVE"`, or `"VMX"`, or `"DOCKER"`.  I think the first parameter should be renamed for added clarity.  Perhaps `description string`?

Reply:
Yeah, it's kind of section, but I suppose I can use some better param name

Reply:
fixed in f7ac492